taprpc/universerpc/universe.proto

```diff
@@ -19,6 +19,8 @@ service Universe {
     */
     rpc QueryAssetRoots (AssetRootQuery) returns (QueryRootResponse);
 
+    rpc DeleteAssetRoot (AssetRootQuery) returns (QueryRootResponse);
```

Comment:
We should create and use a new message for the response, otherwise things might get awkward in the future (and we might need to care about backward compatibility then).

Also, needs a documentation comment.